accumulator_updater/programs/accumulator_updater/src/lib.rs

```diff
@@ -153,17 +181,19 @@ pub struct Initialize<'info> {
     pub system_program: Program<'info, System>,
 }
 
+
 #[derive(Accounts)]
-pub struct AddAllowedProgram<'info> {
+pub struct UpdateWhitelist<'info> {
     #[account(mut)]
-    pub payer:          Signer<'info>,
+    pub payer:     Signer<'info>,
+    pub authority: Signer<'info>,
     #[account(
     mut,
```

Comment:
Nitpick: I don't know if rustfmt can/should be doing this but a single extra indent here makes sense.

Reply:
it seems like based on the below github issues/PRs,  rustfmt should've handled this automatically but not sure why it didn't. I manually indented it though. thanks for the catch

https://github.com/rust-lang/rustfmt/issues/2423
https://github.com/rust-lang/rustfmt/pull/2809/files#diff-e47dc5ac75eba9b921cbcd39a55310f28c152f14f38e5b496b3b8c3f805ac0e5

Reply:
Nah that's ok, intentionally not formatting comments because when you actually want to add structure it just craps over everything. There's a comment above I mentioned but It's good to manually keep formatting in comments good because we can't rely on rustfmt